test/git-open.bats

```diff
@@ -269,6 +269,35 @@ setup() {
 
 }
 
+
+@test "bitbucket server with different root context" {
```

Comment:
**TYPO:** Please add colon here (`bitbucket: `) so the CI output looks nice. Same on the following unit test.

Reply:
Ah yes, thanks for noticing this. I also saw that some of the basic Bitbucket Server tests were duplicated in an earlier commit - I'll fix this at the same time as the naming...